types/thumbsupply/thumbsupply-tests.ts

```diff
@@ -0,0 +1,77 @@
+import thumbsupply from 'thumbsupply';
```

Comment:
Please use the corresponding `import thumbsupply = require('thumbsupply');` syntax here and remove the `"allowSyntheticDefaultImports": true` from the `tsconfig.json` below. We usually want to avoid having extraneous compiler options in your `tsconfig.json` in DT, since it makes it easier for us to maintain the many DT packages.

Reply:
Thank you for your new review.
**There is therefore no way to import this module as an ES module ?**

I have read that declaring an empty namespace `declare namespace thumbsupply {};` is strongly discouraged and that the preferred solution is the one i choose :
![Screenshot from 2022-09-29 09-53-07](https://user-images.githubusercontent.com/27495288/192973764-e5c65865-41ca-4f64-8709-06c29600439e.png)

I have the feeling that this import style is needed to work in a project with the recommended rules from @typescript-eslint/eslint-plugin (no-var-require rule).

Thank you for your help.